libsrc/creativision/libref.s

```diff
@@ -0,0 +1,9 @@
+;
+; Oliver Schmidt, 2013-05-31
+;
+
+        .export         joy_libref, tgi_libref
+        .import         _exit
+
+joy_libref      :=      _exit
+tgi_libref      :=      _exit
```

Comment:
Right now there's no JOY nor TGI driver. I understand that both are reasonable for this target. Do you have actual plans on them?

To be honest I'm afraid that after this pull request is merged nobody will care for this target. Therefore I'm wondering if

- The joystick code in joy.s shouldn't be reworked as JOY driver right in the first place
- The tgi_libref above removed until someone actually writes a TGI driver